plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -228,7 +228,7 @@ public void testDropSchema()
     public void testDropSchemaNameClash()
             throws Exception
     {
-        String schemaName = "Test_Drop_Case_Sensitive_Clash";
+        String schemaName = "Test_Drop_Case_Sensitive_Clash_" + randomTableSuffix();
```

Comment:
These schemas needs to be purged as well (ok to follow up)

Reply:
See `withSchema` impl (to which these are passed). If `withSchema` misses these the `cleanUpDatasets` test will remove these.